tests/test_chromatic.py

```diff
@@ -1201,6 +1225,269 @@ def test_centroid():
+    im_interp = im_exact.copy()
+    im_interp = interp_psf_1.drawImage(bandpass, image=im_interp, scale=scale)
+    expected_flux = disk_SED.calculateFlux(bandpass)
+    frac_diff_exact = abs(im_exact.array.sum()/expected_flux-1.0)
+    frac_diff_interp = abs(im_interp.array.sum()/expected_flux-1.0)
+    np.testing.assert_almost_equal(
+        frac_diff_exact, 0., decimal=3,
+        err_msg='InterpolatedChromaticObject flux is wrong (exact calculation)')
+    np.testing.assert_almost_equal(
+        frac_diff_interp, 0.0, decimal=3,
+        err_msg='InterpolatedChromaticObject flux is wrong (interpolated calculation)')
+
+    # Check that when an InterpolatedChromaticObject is convolved with a ChromaticObject, the flux
+    # that we get is correct.  (Note, we don't have to try convolving it with something achromatic,
+    # because the test above does this implicitly - the pixel response that it gets convolved with
+    # is achromatic.)  We are forcing it to go through the evaluation of
+    # ChromaticConvolution.drawImage().
+    gal = galsim.Exponential(half_light_radius = 2.*scale)
+    gal = gal.shear(g2 = 0.3)
+    gal = disk_SED*gal
+    obj_exact = galsim.Convolve(exact_psf, gal)
+    obj_interp = galsim.Convolve(interp_psf, gal)
+    im_exact = obj_exact.drawImage(bandpass_g, scale=scale)
+    im_interp = im_exact.copy()
+    im_interp = obj_interp.drawImage(bandpass_g, image=im_interp, scale=scale)
+    expected_flux = disk_SED.calculateFlux(bandpass_g)
+    frac_diff_exact = abs(im_exact.array.sum()/expected_flux-1.0)
+    frac_diff_interp = abs(im_interp.array.sum()/expected_flux-1.0)
+    np.testing.assert_almost_equal(
+        frac_diff_exact, 0.0, decimal=3,
+        err_msg='InterpolatedChromaticObject flux is wrong when convolved with ChromaticObject'
+        ' (exact calculation)')
+    np.testing.assert_almost_equal(
+        frac_diff_interp, 0.0, decimal=3,
+        err_msg='InterpolatedChromaticObject flux is wrong when convolved with ChromaticObject'
+        ' (interpolated calculation)')
+    np.testing.assert_array_almost_equal(
+        im_interp.array, im_exact.array, decimal=4,
+        err_msg='InterpolatedChromaticObject results differ for interpolated vs. exact'
+        ' when convolving with ChromaticObject')
+
+    # Check that when an InterpolatedChromaticObject is convolved with a ChromaticSum, the flux
+    # that we get is correct.
+    bulge = galsim.DeVaucouleurs(half_light_radius = 1.5*scale)
+    bulge = bulge_SED*bulge
+    tot_gal = gal + bulge
+    obj_exact = galsim.Convolve(exact_psf, tot_gal)
+    obj_interp = galsim.Convolve(interp_psf, tot_gal)
+    im_exact = obj_exact.drawImage(bandpass, scale=scale)
+    im_interp = im_exact.copy()
+    im_interp = obj_interp.drawImage(bandpass, image=im_interp, scale=scale)
+    expected_flux = disk_SED.calculateFlux(bandpass) + bulge_SED.calculateFlux(bandpass)
+    expected_flux *= (1.0 - galsim.GSParams().alias_threshold)
```

Comment:
Using alias_threshold here seems like a weird thing to do to me since image corners probe larger radii than that which encloses (1-alias_threshold).  Tests pass on my system with this line commented out anyway.